openadapt/share.py

```diff
@@ -0,0 +1,120 @@
+"""Copy a recording from one computer to another.
+
+Usage:
+    python -m openadapt.share send --recording_id=1 
+    python -m openadapt.share receive <wormhole_code>
+"""
+
+from zipfile import ZipFile, ZIP_DEFLATED
+import os
+import subprocess
+
+from loguru import logger
+import fire
+
+from openadapt import config, db, utils
+
+
+LOG_LEVEL = "INFO"
+utils.configure_logging(logger, LOG_LEVEL)
+
+
+def export_recording_to_folder(recording_id: int) -> None:
+    """Export a recording to a zip file.
+
+    Args:
+        recording_id (int): The ID of the recording to export.
+
+    Returns:
+        str: The path of the created zip file.
+    """
+    # TODO: replace with alternative, e.g.:
+    # - https://stackoverflow.com/questions/28238785/sqlalchemy-python-multiple-databases
+    # - https://github.com/bigbag/sqlalchemy-multiple-db
+    recording_db_path = db.export_recording(recording_id)
+
+    if recording_db_path:
```

Comment:
Is this necessary?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/223